Gemfile

```diff
@@ -72,14 +72,17 @@ group :development, :docker, :test do
   gem 'rspec-rails'
 end
 
+group :docker, :test do
```

Comment:
Would it make sense to add `:development` to this group? I'm not sure how we'd use it from the rails console, since we wouldn't want to commit code with `binding.pry` in it... Hmm, maybe just ignore this thought then.  :P